apps/chat-e2e/src/tests/promptFoldersSpecialChars.test.ts

```diff
@@ -0,0 +1,209 @@
+      'EPMRTC-2975',
+      'EPMRTC-2976',
+      'EPMRTC-2977',
+      'EPMRTC-2978',
+      'EPMRTC-2979',
+      'EPMRTC-2980',
+      'EPMRTC-2981',
+      'EPMRTC-2982',
+    );
+    const folderName = ExpectedConstants.newFolderWithIndexTitle(1);
+    const newNameWithEndDot = `${folderName}.`;
+    const newNameWithSpecialChars = `${folderName} ${ExpectedConstants.allowedSpecialChars}`;
+    const nameWithRestrictedChars = `Folder${ExpectedConstants.restrictedNameChars}_name`;
+    const expectedFolderName = 'Folder_name';
+    const newNameWithSpaces = 'Folder   1';
+    const expectedName = 'Folder with spaces';
+    const nameWithSpacesBeforeAndAfter = `   ${expectedName}   `;
+    const newNameWithEmojis = '😂👍🥳 😷 🤧 🤠 🥴😇 😈 ⭐あおㅁㄹñ¿äß';
+
+    await dialTest.step('Create prompt folder', async () => {
+      await dialHomePage.openHomePage();
+      await dialHomePage.waitForPageLoaded();
+
+      await promptBar.createNewFolder();
+      await promptBarFolderAssertion.assertFolderState(
+        { name: folderName },
+        'visible',
+      );
+    });
+
+    await dialTest.step(
+      'Rename it to have a dot at the end of the name',
+      async () => {
+        await folderPrompts.openFolderDropdownMenu(folderName);
+        await folderDropdownMenu.selectMenuOption(MenuOptions.rename);
+        await folderPrompts.editFolderName(newNameWithEndDot);
+      },
+    );
+
+    await dialTest.step('Click on confirmation button', async () => {
+      await folderPrompts.getEditFolderInputActions().clickTickButton();
+
+      const errorMessage = await errorToast.getElementContent();
+      expect
+        .soft(errorMessage, ExpectedMessages.notAllowedNameErrorShown)
+        .toBe(ExpectedConstants.nameWithDotErrorMessage);
+
+      // Verify folder name stays in edit mode
+      await expect
+        .soft(
+          folderPrompts.getEditFolderInput().getElementLocator(),
+          ExpectedMessages.folderEditModeIsActive,
+        )
+        .toBeVisible();
+
+      // Closing the toast to move forward
+      await errorToast.closeToast();
+    });
+
+    await dialTest.step('Rename it to contain special characters', async () => {
+      await folderPrompts.editFolderNameWithTick(newNameWithSpecialChars);
+      await promptBarFolderAssertion.assertFolderState(
+        { name: newNameWithSpecialChars },
+        'visible',
+      );
+    });
+
+    await dialTest.step(
+      'Try to type restricted special characters',
+      async () => {
+        await folderPrompts.openFolderDropdownMenu(newNameWithSpecialChars);
+        await folderDropdownMenu.selectMenuOption(MenuOptions.rename);
+        for (const char of ExpectedConstants.restrictedNameChars.split('')) {
+          await folderPrompts.editFolderName(char);
+          expect
+            .soft(
+              await folderPrompts.getEditFolderInput().getEditInputValue(),
+              ExpectedMessages.charactersAreNotDisplayed,
+            )
+            .toBe('');
```

Comment:
let's move this verification to FolderAssertion to have smth like:

`await promptBarFolderAssertion.assertFolderEditInputValue('');`

Reply:
done